src/pybamm/parameters/parameter_values.py

```diff
@@ -417,6 +418,16 @@ def set_initial_ocps(
         return parameter_values
 
     @staticmethod
+    @deprecate_arguments(
+        deprecated_args={
+            "electrode diffusivity": "Use 'particle diffusivity' instead.",
+            "1 + dlnf/dlnc": "Use 'Thermodynamic factor' instead.",
+        },
+        deprecated_in="24.1.0",
+        removed_in="25.3.0",
+        current_version="25.1.0",
+        msg="Update your parameter names to avoid future compatibility issues.",
+    )
```

Comment:
Could we have multiple messages per argument? The current message here is not informative enough, and is rather a step back in comparison to the messages that currently exist already. That is, it would be nice if `@deprecate_arguments()` could take a dictionary, i.e., accept a key-value pair for an argument and its corresponding deprecation message. The current approach you've taken doesn't seem to accommodate this behaviour.